readme.md

```diff
@@ -41,6 +41,7 @@ Please check the <a href="https://github.com/tobiasbueschel/awesome-pokemon/blob
 #### APIs
 - [PokeAPI](https://github.com/PokeAPI/pokeapi/) - A RESTful API for Pokémon (http://pokeapi.co/).
 - [pgoapi](https://github.com/tejado/pgoapi) - Pokémon GO API lib/demo.
+- [pokemon-API](https://github.com/Purukitto/pokemon-api) - Beginner friendly API endpoint that returns Pokemon Data in JSON format (https://app.pokemon-api.xyz)
```

Comment:
Thanks for the PR - happy to merge it, but could you correct the spelling of Pokémon first and add a full stop to the end of the sentence?

Have a great weekend!